docs/develop/api-reference/sorted-set-collections.md

```diff
@@ -0,0 +1,177 @@
+
+</details>
+
+### SortedSetFetch
+
+Fetch all elements in a sorted set and return them in ascending or descending order.
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| cacheName       | String          | Name of the cache.                            |
+| setName         | String          | Name of the sorted set item. |
+| numberOfResults | Optional[integer]   | How many results to return, default is all results. |
+| order           | Ascending \| Descending | The order you want the sorted set returned |
+
+<details>
+  <summary>Method response object</summary>
+
+* Hit
+    * elements(): SortedSetElement[]
+* Miss
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### SortedSetGetScore
+
+Gets an existing element and its associated score from the sorted set.
+
+| Name             | Type                | Description                                   |
+| ---------------- | ------------------- | --------------------------------------------- |
+| cacheName        | String              | Name of the cache.                            |
+| setName          | String              | Name of the sorted set item. |
+| elements         | String[] \| bytes[] | Element in the sorted set to be get the score of. |
+
+<details>
+  <summary>Method response object</summary>
+
+* Cache hit
+    * Elements() (returns hit/miss per element)
+      * Hit:
+        * Score: number
+      * Miss
+* Cache miss (if the sorted set does not exist)
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### SortedSetRemove
+
+Removes an element from a sorted set.
+
+| Name            | Type             | Description                                   |
+| --------------- | ---------------- | --------------------------------------------- |
+| cacheName       | String           | Name of the cache.                            |
+| setName         | String           | Name of the set item to be altered. |
+| elements        | All \| string[] \| bytes[] | Element to be removed by this operation. |
+
+You can remove either elements all elements or a set of elements.
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### SortedSetGetRank
+
+What position is this element in the specificed sorted set?
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| cacheName       | String          | Name of the cache.                            |
+| setName         | String          | Name of the set item to be altered. |
+| element         | String \| bytes | Element to be removed by this operation. |
```

Comment:
This description isn't right, should be `Name of the element to retrieve rank` or something similar